src/icon_exclaim_perf_tools/db/schema.py

```diff
@@ -101,7 +106,7 @@ def mode(self):
 
     experiment: sqla.orm.Mapped[str]
     jobid: sqla.orm.Mapped[int] = sqla.orm.mapped_column(nullable=True, unique=True)
-    mode = sqla.orm.mapped_column(enum_field(ModelRunMode))
+    mode = sqla.orm.mapped_column(enum_field(ModelRunMode), nullable=True)
```

Comment:
```suggestion
    mode = sqla.orm.mapped_column(enum_field(ModelRunMode))
```
@kotsaloscv I reverted this change as it doesn't seem necessary. Let me know if I overlooked something.